### DISCUSSION TITLE
Possible to ignore heading on render?

### DISCUSSION THREAD

**Discussion by erauchway** [quarto-dev/quarto-cli#3573] [Q&A]
Hi; I'm curious to know if there a Quarto equivalent of the :ignore: tag in org-mode, which will allow one to use a header for organizational purposes but prevent it from being rendered on export in LaTeX or docx? 

(discussion of the org-mode tag below)

https://emacs.stackexchange.com/questions/38184/org-mode-ignore-heading-when-exporting-to-latex

**Comment by cscheid** [comment]
Take a look at https://quarto.org/docs/authoring/conditional.html.

You can use `content-hidden` without a clause and that will not render the content.

**Reply by erauchway** [reply]
Thank you! Possibly I'm doing this wrong, but this:

````
---
title: "Test of Reveal.js"
format: 
  revealjs: default
  docx: default
---

## Here is a slide 
It's got some text on it.

::: {.content-visible when-format="html"}

## Title will only appear in HTML

:::

This text is confusing the machinery.

## Here is another slide 

It's got some text on it too

````

ends up with the line "This text is confusing the machinery" appearing on every slide in the reveal.js presentation. Did I use the conditional formatting incorrectly?

**Reply by cscheid** [reply]
Nope, this looks like a quarto bug.

**Reply by cscheid** [reply]
Tracking it [here](https://github.com/quarto-dev/quarto-cli/issues/3575).

**Reply by erauchway** [reply]
Thanks!
